### PR TITLE
feat(API): fix of credentials

### PR DESCRIPTION
**Pull request by riteshsonawane1372**
# Tasks / Issue :construction: 🔧 

- [x] #17 

# Solution :heavy_check_mark:

Save Aws credentials. 

# Note to reviewers :notebook:

### CODE REVIEW

**Comment by dipankardas011** [comment]
I think this dev branch is not having the expected commits may be fault by my side

**Comment by dipankardas011** [comment]
@riteshsonawane1372 fixed the dev branch may be you need to rebase you commits

**Comment by riteshsonawane1372** [comment]
> @riteshsonawane1372 fixed the dev branch may be you need to rebase you commits

image

File: api/aws/main.go [hunk 1]
```diff
@@ -23,32 +23,30 @@ func fetchAPIKey() string {
...
+	logger.Print("Enter your ACCESS-KEY: ")
+	accesKey, err := util.UserInputCredentials(logger)
+	if err != nil {
+		panic(err.Error())
+	}
+
+	logger.Print("Enter your SECRET-KEY: ")
+	secret, err := util.UserInputCredentials(logger)
+	if err != nil {
+		panic(err.Error())
```
Comment (dipankardas011) [inline api/aws/main.go:37]: you can put here logger.Err and return false

File: api/aws/main.go [hunk 2]
```diff
@@ -23,32 +23,30 @@ func fetchAPIKey() string {
...
-	// if err != nil {
-	// 	fmt.Println(err.Error())
-	// 	return false
-	// }
+func Credentials(logger log.Logger) bool {
+
+	logger.Print("Enter your ACCESS-KEY: ")
+	accesKey, err := util.UserInputCredentials(logger)
+	if err != nil {
+		panic(err.Error())
```
Comment (dipankardas011) [inline api/aws/main.go:31]: same here as well

**Review by dipankardas011 — CHANGES_REQUESTED** [review]

**Review by dipankardas011 — APPROVED** [review]

**Comment by dipankardas011** [comment]
@riteshsonawane1372 do fill the solution part of this PR description so that we can merge

**Comment by dipankardas011** [comment]
@riteshsonawane1372 also if you can do rebase with latest dev branch

**Comment by riteshsonawane1372** [comment]
> @riteshsonawane1372 also if you can do rebase with latest dev branch

I wasn't able to rebase the branch, and I would greatly appreciate any assistance you can provide.

**Review by dipankardas011 — APPROVED** [review]

**Comment by dipankardas011** [comment]
@AvineshTripathi it's wierd that automerge not working here

**Comment by dipankardas011** [comment]
@riteshsonawane1372 thanks for creating this pr 🙏

**Comment by AvineshTripathi** [comment]
> @AvineshTripathi it's wierd that automerge not working here

we should debug!

**Comment by riteshsonawane1372** [comment]
> @riteshsonawane1372 thanks for creating this pr 🙏

 If possible, create a new branch "dev-aws" from main

**Pull request merged**